guides/doc-Provisioning_Guide/topics/proc_installing_ipv6_environment.adoc

```diff
@@ -0,0 +1,264 @@
+
+The DHCPv6 server must be configured to return `http://{smartproxy-example-com}:8000/EFI/grub2/grubx64.efi` URL to clients with vendor class string "HTTPClients". The URL must point to either {ProjectServer} or {SmartProxy} with TFTP and HTTPBoot features turned on. Example configuration for ISC DHCP server from {RHEL}:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# cat /etc/dhcp/dhcpd6.conf
+default-lease-time 2592000;
+preferred-lifetime 604800;
+option dhcp-renewal-time 3600;
+option dhcp-rebinding-time 7200;
+allow leasequery;
+option dhcp6.info-refresh-time 21600;
+option dhcp6.vendor-class-identifier code 16 = string;
+option dhcp6.vendor-class code 16 = {integer 32, integer 16, string};
+option dhcp6.user-class code 15 = string;
+option dhcp6.bootfile-url code 59 = string;
+option dhcp6.client-arch-type code 61 = array of unsigned integer 16;
+
+subnet6 2001:db8::/64 {
+  range6 2001:db8::aaaa 2001:db8::ffff;
+  option dhcp6.name-servers 2001:db8::2;
+
+  if exists dhcp6.user-class and
+    substring(option dhcp6.user-class, 2, 4) = "iPXE" {
+    # iPXE script file location
+    option dhcp6.bootfile-url "http://{smartproxy-example-com}:8000/unattended/iPXE";
+  } else if option dhcp6.client-arch-type = 00:06 {
+    # UEFI x86 PXE/TFTP boot via IPv6
+    option dhcp6.bootfile-url "tftp://{smartproxy-example-com}:8000/ipxe32.efi";
+  } else if option dhcp6.client-arch-type = 00:07 {
+    # UEFI x86-64 PXE/TFTP boot via IPv6
+    option dhcp6.bootfile-url "tftp://{smartproxy-example-com}:8000/grub2/grubx64.efi";
+  } else if option dhcp6.client-arch-type = 00:0f {
+    # UEFI x86 HTTP boot via IPv6
+    option dhcp6.vendor-class 0 10 "HTTPClient";
+    option dhcp6.bootfile-url "http://{smartproxy-example-com}:8000/EFI/grub2/grubx64.efi";
+  } else if option dhcp6.client-arch-type = 00:10 {
+    # UEFI x86-64 HTTP boot via IPv6
+    option dhcp6.vendor-class 0 10 "HTTPClient";
+    option dhcp6.bootfile-url "http://{smartproxy-example-com}:8000/EFI/grub2/grubx64.efi";
+  } else {
+    # Support a hypothetical BIOS system that can PXE boot over IPv6
+    option dhcp6.bootfile-url "tftp://{smartproxy-example-com}:8000/undionly.kpxe";
+  }
+}
+----
+
+Example configuration for Router Advertisment server from {RHEL}:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# cat /etc/radvd.conf
+interface eth0
+{
+        AdvSendAdvert on;
+        AdvManagedFlag on;
+        AdvOtherConfigFlag on;
+
+        prefix 2001:db8::/64
+        {
+                AdvAutonomous off;
+        };
+
+        RDNSS 2001:db8::2 {
+        };
+};
+----
+
+NOTE: The DNS server must resolve the hostname in bootfile URL, `{smartproxy-example-com}` in the example above. It must also match DNS server set in the Subnet.
+
+== Importing provisioning templates
+
+NOTE: This should be removed for Foreman 2.0 or Satellite 6.8.
```

Comment:
Luckily these are now part of all supported releases :)

Reply:
Removed.